documents/tutorials/contiv-policy.html

```diff
@@ -605,6 +605,192 @@
 
 </code></pre>
 
+<h3>Chapter 3 - Bandwidth Policy</h3>
+<p>In this chapter, we will explore bandwidth policy feature of contiv. 
+We will create tenant, network and groups. Then we will attach networkprofile to one group
+and verify that applied badwidth is working or not as expected in data path.</p>
+<p>So, let us create tenant and network and group A under network.</p>
```

Comment:
So, let us create tenant and network and group A under network. -> So, let us create a tenant, a network and a group "A" under the network.